packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md

```diff
@@ -16,3 +16,19 @@ Add the following endpoint URLs for the Okta Identity Provider that you are conf
 * **Userinfo endpoint (optional)**: The endpoint for getting identity information about the user. For example: `https://${theOktaIdPOrg}/oauth2/v1/userinfo`
 
 > **Note:** Okta requires an access token returned from the Identity Provider if you add the `/userinfo` endpoint URL.
+
+In the optional **Authentication Settings** section:
+
+* **IdP Username:** This is the expression (written in Okta Expression Language) that is used to convert an Identity Provider attribute to the application user's username. This Identity Provider username is used for matching an application user to an Okta User.
+
+    For example, the value `idpuser.email` means that it takes the email attribute passed by the Identity Provider and maps it to the Okta application user's username property.
+
+    You can enter an expression to reformat the value, if desired. For example, if the social username is `john.doe@mycompany.com`, then you could specify the replacement of `mycompany` with `endpointA.mycompany` to make the transformed username `john.doe@endpointA.mycompany.com`. See [Okta Expression Language](/docs/reference/okta-expression-language/).
+
+    **Note:** When you use multi-tenancy, we recommend that you add a unique username format with a suffix per spoke org to prevent all potential impersonation, except with intentional scenarios such as using AD as the sign-in source for Okta.
+
+* **Filter > Only allow usernames that match defined RegEx Pattern:** Select this option to only authenticate users with transformed usernames that match a regular expression pattern in the text field that appears. This filters the IdP username to prevent the IdP from authenticating unintended or privileged users. Users are only authenticated if the transformed username matches the regular expression pattern.
```

Comment:
```suggestion
* **Filter > Only allow usernames that match defined RegEx Pattern:** Select this option to only authenticate users with transformed usernames that match a regular expression pattern in the text field that appears. This filters the IdP username to prevent the IdP from authenticating unintended users. Users are only authenticated if the transformed username matches the regular expression pattern.
```